### PR TITLE
Setup proguard

### PR DESCRIPTION
**Pull request by ivacf**

### CODE REVIEW

File: app/proguard-rules.pro [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Add project specific ProGuard rules here.
 # By default, the flags in this file are appended to flags specified
-# in /Users/hitherejoe/Android Studio.app/sdk/tools/proguard/proguard-android.txt
```
Comment (jsonfry) [inline app/proguard-rules.pro]: ew

**Comment by jsonfry** [comment]
Code LGTM, is there any further testing you think we should do before merging?

**Comment by jsonfry** [comment]
Although, do we want to include a release key?

**Comment by jsonfry** [comment]
As this is public :)

**Comment by ivacf** [comment]
We are never gonna use the release key. This key must change when we set up a new project.  I just added it so I could do a release build and test proguard.

**Comment by jsonfry** [comment]
I think it would be a good idea therefore to not include it, so that it definitely does get replaced by us (we'll forget, I know I would), and when other people use it, it's not confusing for them to have a release keystore they don't know the password to, and shouldn't use anyway. What do you think? Am I crazy?

**Comment by ivacf** [comment]
@jsonfry release key removed

**Pull request merged**